test/model/finders_test.rb

```diff
@@ -45,6 +56,13 @@ class Human
       assert !Human.exist?('not_an_id')
       assert Human.exist?(@human1.id)
     end
+
+    test 'check existence with given quorum' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.